dbms/src/test/java/org/polypheny/db/sql/fun/StringFunctionsTest.java

```diff
@@ -288,7 +284,7 @@ public void subString() throws SQLException {
     }
 
 
-    @Ignore
+    @Disabled
```

Comment:
Can you enable this test?

Reply:
Added missing Functions method, enabled.